mkat_fpga_tests/test_cbf.py

```diff
@@ -720,4 +720,5 @@ def test_vacc(self):
             # Check that the complexised response is equal to the expected response
             Aqf.is_true(np.array_equal(expected_response, response),
                 'Check that the complexised response is equal'
-                    ' to the expected response')
```

Comment:
I think 'complexise' is a bit of internal slang, perhaps while you are at it remove the 'complexised' and replace it with 'check that the accumulator response is equal ...'